src/components/services/SpeedtestTracker.vue

```diff
@@ -0,0 +1,86 @@
+<template>
```

Comment:
Please extend the Generic service and modify only the #content / subtitle portion